### PR TITLE
update instructions for codeless connector

### PR DESCRIPTION
**Pull request by austinmccollum**
Update instruction for codeless connector validation since the DataConnector_API_CCP_template.json won't work out of the box until you reduce it to just the *connectorUiConfig* collection.

Tested each template with the validation tool, and only the CCP template fails. Determined the template will validate when reducing to just the connectorUiConfig collection.

Reason to change - I want to be able to point to this validation tool from the codeless connector learn/docs page.

### CODE REVIEW

**Comment by v-atulyadav** [comment]
Hi @v-vdixit, please provide your feedback. Thanks

**Comment by v-vdixit** [comment]
Hi @austinmccollum we are looking into this, thanks!

**Comment by v-vdixit** [comment]
Hi @austinmccollum we are still working on this, thanks!

File: DataConnectors/ReadMe.md [hunk 1]
```diff
@@ -180,7 +180,7 @@ Once you have a working POC, you are ready to build, validate the data connector
 3. **Validate the Connector UX** – Follow these steps to render and validate the connector UX you just built
     1.	The test utility can be accessed by this URL - https://aka.ms/sentineldataconnectorvalidateurl
     2.  Go to Microsoft Sentinel -> Data Connectors 
-    3.	Click the “import” button and select the json file you created as follows.
+    3.	Click the "import" button and select the json file you created as follows. To validate a codeless connector with this tool, be sure to only upload a json file containing the `connectorUiConfig` collection.
```
Comment (v-vdixit) [inline DataConnectors/ReadMe.md:183]: Hi @austinmccollum Can you please update the text in accordance with the Azure documentation - [Create Codeless Connector](https://github.com/MicrosoftDocs/azure-docs/blob/main/articles/sentinel/create-codeless-connector.md), to ensure consistency, adding a screenshot for the same for your reference, thanks!
image

Comment (v-vdixit) [reply DataConnectors/ReadMe.md:183]: Hi @austinmccollum thanks for letting us know, we will discuss with the team and update you shortly.

**Review by v-vdixit — APPROVED** [review]

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**